frontend/src/components/ModalContent.vue

```diff
@@ -1,119 +1,116 @@
 <template>
   <v-card>
-    <v-card-title class="title-header">
-      <v-row>
-        <p class="modal-title">{{ selectedProject.name }}</p>
-        <v-card-actions>
-          <v-btn
-            class="mr-2"
-            absolute
-            fab
-            rounded
-            icon
-            large
-            right
-            @click="onClick()"
-          >
-            <v-icon size="3.5rem" color="white">close</v-icon>
-          </v-btn>
-        </v-card-actions>
-      </v-row>
+    <v-card-title class="container-title unified-padding">
+      <h1>{{ selectedProject.name }}</h1>
+      <div>
+        <v-btn class="hidden-xs-only" rounded icon large @click="onClick()">
+          <v-icon size="3.5rem" color="white">close</v-icon>
```

Comment:
would be better to pass it as some variables not hardcoded like that, if not pass color as hex (#ffffff) not `white`
you can try this approach: 
https://stackoverflow.com/a/61378302

Reply:
Ok. so the options in the link regard the situation when I actually want to change sth about default vuetify settings. Given that I realize how fucked up it is to have part of the styling through vuetify props and others through regular classes is there actually a reason to try to overwrite that? I mean veutfy is what it is, but isn't it overkill to write css just to override the default settings to the same effect? Please straight me out if I'm wrong.

Reply:
**Final changes:**
1. I dropped v-btn and v-icon altogether (in this case vuetify does more harm than good)
2. I created global classes for two kinds of buttons which use globally defined colors